<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Student Schedules</string>
  </property>
  <widget class="QTabWidget" name="MenuTabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>981</width>
     <height>581</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="StudentsInput">
    <attribute name="title">
     <string>Students Input</string>
    </attribute>
    <widget class="Line" name="VerticalManualInputLine">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>-50</y>
       <width>20</width>
       <height>691</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="UploadExcelText">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Upload Excel File &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="ManualInputText">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Manual Student Input&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="ImportantNote">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#aa0000;&quot;&gt;***Please ensure proper excel file format ***&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#aa0000;&quot;&gt;as shown below&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="HoritzontalLine">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>20</y>
       <width>1161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>500</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RevertButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>500</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Revert Changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UploadFileButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>500</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload Excel File</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>451</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="TermNum">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Select Term Number</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="TermInput">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="BCOM">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Business Communications (BCOM)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="BCOMinput">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="PCOM">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Professional Communication (PCOM)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="PCOMinput">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="PM">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Project Management (PM)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="PMinput">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="BA">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Business Analysis (BA)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="BAinput">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="GLM">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Global Logistics Management (GLM)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="GLMinput">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="FS">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Full Stack Web Development (FS)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="FSinput">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="DXD">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Digital Experience Design Foundation (DXD)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="DXDinput">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="BK">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Book Keeping Certificate (BK)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="BKinput">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="SCM">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>Supply Chain Management (SCM)</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QSpinBox" name="SCMinput">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Cohorts">
    <attribute name="title">
     <string>Cohorts</string>
    </attribute>
    <widget class="QTableWidget" name="CohortTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>911</width>
       <height>501</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>BCOM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PCOM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>BA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>GLM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FSS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DXD</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>BK</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SCM</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="SaveAsCohort">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>520</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save as (excel file)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Schedules">
    <attribute name="title">
     <string>Schedule</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
